webpack/webpack.config.js

```diff
@@ -14,10 +15,10 @@ module.exports = (args = { mode: "production", analyze: false }) => {
         entry: {
             areaLanding: {
                 import: path.resolve(__dirname, "../public/ts/area-landing.ts"),
-                dependOn: "mapboxGL"
+                // dependOn: "mapboxGL"
```

Comment:
Is it worth removing these comments altogether

Reply:
Yep, these will all be added back in at a later date hopefully